flink-runtime-web/web-dashboard/src/app/pages/job/checkpoints/job-checkpoints.component.html

```diff
@@ -264,6 +264,10 @@
             <td>Tolerable Failed Checkpoints</td>
             <td>{{ checkPointConfig['tolerable_failed_checkpoints'] }}</td>
           </tr>
+          <tr>
+            <td>Alignment timeout</td>
+            <td>{{ checkPointConfig['alignment_timeout'] }}</td>
+          </tr>
```

Comment:
I ran a job locally and observed that:
1. Timeout is shown if UC are disabled (in which case it doesn't make any sense). Do you think we should/could hide it? Or maybe show a relevant portion from docs (`ExecutionCheckpointingOptions`), for example as a tooltip?
1. Same if it is zero. 
1. Timeout is shown as a last line. Could you make it next to "unaligned checkpoints"?
1. Timeout is shown without a unit. Could you add a unit similar to checkpoint_timeout?

Reply:
1. I think showing the docs from ExecutionCheckpointingOptions is a good idea but it's better to do it in siperated ticket. But in this ticket, I agree that we can hide the timeout if UC is disabled
2. I don't think that we should hide the field if it has value zero(if UC is enabled) because it is better to see that value is zero than think why you don't see this field because it is zero or because it isn't supported
3. done
4. done